custom/dhis2/payload_generators.py

```diff
@@ -68,6 +66,7 @@ def get_payload(self, repeat_record, form):
             logger.debug('DHIS2: Processing Register Child form')
             push_case(case, dhis2_api)
             # We just need to enroll. No event to create
+            raise IgnoreDocument
```

Comment:
Looking at #5803 I think lines 74 and 90 should also raise `IgnoreDocument` exceptiosn